### PR TITLE
Download homework

### PR DESCRIPTION
**Pull request by kostuch76**

### CODE REVIEW

File: homework/linux.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ 1353  history | tail -397 > commands.txt 
+ 1354  history | tail -39 > commands.txt 
+ 1355  cat commands.txt 
+ 1356  history | tail -42 > commands.txt 
+ 1357  cat commands.txt 
+ 1358  history | tail -452 > commands.txt 
+ 1359  history | tail -46 > commands.txt 
+ 1360  cat commands.txt 
+ 1361  history
+ 1362  history | tail -50 > commands.txt
```
Comment (nauka-programowania-MA) [inline homework/linux.txt:73]: add switch also -n :)

**Comment by ziobron** [comment]
Your solution is correct, but our workflow is not ;) 
It should be accepted. Can you raise a new Issue about it and link #93 in there? You'll get an extra 2 XP for that. I will fix it in a few days.

**Comment by ziobron** [comment]
✅ CI Passed - Task: linux
❌ 10 XP not granted. Probably you don't have an account on the old course platform. Please register on https://edu.coders.school and ask @ziobron to take a look here.

**Comment by ziobron** [comment]
✅ CI Passed - Task: linux
🏅 Granted 10 XP for @kostuch76

**Review by ziobron — APPROVED** [review]

**Pull request closed without merging**